server/fakedb/Schedule.csv

```diff
@@ -0,0 +1,22 @@
+﻿eventId,start,end,locationLatitude,locationLongitude,repeat
```

Comment:
Should the start / end field be more specific than just dates? Are there time slots?

Reply:
Even though the current mock data don't have time info, we can write dates with time in start / end field.

Reply:
Actually start and end columns are DateTime type. It contains Time information.